CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and follows a [Backwards Compatability Policy](https://docs.solana.com/developing/backwards-compatibility)
+
+## Adding to this Changelog
+### Audience
+* Entries in this log are intended to be easily understood by consensus validators, rpc operators, and dapp developers.
+
+### Noteworthy
+* Changes that relate to a SIMD, or add a feature gate should add an entry to this log. Other changes that are relevant to the aforementioned audiences should add an entry at the discretion of authors and reviewers.
+
+### Instructions
+* Update this log with the same commit that implements the change. If the change is spread over several commits update this log with the commit that makes the feature code complete.
+* Add notes to the [Unreleased] section in each branch that you merge to.
+  * Add a description of your change to the Changes section
+  * If your change is likely to require operators to update their configs add to the Upgrade Notes
+* If you add entries on multiple branches please use the same wording if possible. This simplifies the process of diffing between versions of the log. 
+
+## How To Maintain This Changelog
```

Comment:
I was thinking we could first try really only adding to the CHANGELOG in `master` as new features are developed. When a new release branch is created, current contents of CHANGELOG go into the 1.x.0 release nodes and then the file is reset.

We then follow our current manual process for 1.x.y patch release notes for now (maybe consider doing something more later, but baby steps!)

This also reenforces the model that features are not backported (!!), only bug fixes

Reply:
Ok that's reasonable. I've removed the line relating to backports.